devel/ansible/roles/bodhi/files/print-messages.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Print messages sent to the amq.topic exchange
+After=network-online.target
+Wants=network-online.target
+
+[Service]
+User=vagrant
+ExecStart=/usr/bin/fedora-messaging consume --callback="fedora_messaging.example:printer"
```

Comment:
Might be cool to add a .bashrc alias like ```bmessaging``` or something that tails these logs. If you opt for that, don't forget to add it to the motd too ☺